jedi/plugins/stdlib.py

```diff
@@ -592,7 +596,7 @@ def functools_partial(value, arguments, callback):
 
 def functools_partialmethod(value, arguments, callback):
     return ValueSet(
-        PartialMethodObject(instance, arguments)
+        PartialMethodObject(instance, arguments, instance) # FIXME pass correct instance as last arg
```

Comment:
I don't think this is the instance I should be passing, and the last argument really only acts like a boolean at the moment.

Reply:
Just pass nothing :) It should just be `PartialMethodObject(instance, arguments)`. AFAIK that should do it.

Reply:
If I am going to replace the original `PartialObject` I'll have to at least pass a bool to distinguish between the two behaviours. Should I refactor the code with a `is_bool` argument and replace the `is None` checks?

Reply:
Your [previous comment](https://github.com/davidhalter/jedi/pull/1522#issuecomment-598467833) suggested creating an instance parameter in `PartialMethodObject` and passing it to `MergedPartialArguments`, since it had to know about the calling instance. So I assumed we'd have to pass something in there.

Reply:
Yeah, I might not have been clear enough. The moment where the instance is defined is when `py__get__` is called.

Reply:
That might actually explain some of the issues I had while trying to understand the code and get it to work. Still, since I was getting more test failures when omitting the argument I assumed that would be the right track, though.

But I think the test failures might actually be more consistent when I do not pass an instance. The signature tests all start failing, however.

I'll submit another commit with the other changes (refactor, tests), and pass a bool as a placeholder arg for now since that way at least the tests pass. I feel my usefulness on this issue is exhausted, so it probably makes more sense to make this a proper PR and let you figure out the hard parts :)